internal/operators/lvm/manifests_test.go

```diff
@@ -32,6 +34,7 @@ var _ = Describe("LVM manifest generation", func() {
 			}
 
 			_, err = yaml.YAMLToJSON(manifest)
+			fmt.Println(err)
```

Comment:
nit:
you could do `Expect(err).ShouldNot(HaveOccurred(), "yamltojson err: %v", err)`